scenes/quests/template_quests/NO_EDIT/1_NO_EDIT_stealth/NO_EDIT_stealth.tscn

```diff
@@ -62,14 +63,16 @@ position = Vector2(131, 463)
 process_mode = 3
 limit_left = 0
 limit_top = 0
+limit_right = 2816
+limit_bottom = 1280
 position_smoothing_enabled = true
 editor_draw_limits = true
 
 [node name="EnemyGuards" type="Node2D" parent="."]
 y_sort_enabled = true
 
 [node name="Guard1-GoingBackAndForth" parent="EnemyGuards" node_paths=PackedStringArray("patrol_path") instance=ExtResource("6_fg25g")]
-position = Vector2(526, 470)
+position = Vector2(525.9997, 470.00003)
```

Comment:
I notice that Godot editor started doing this since we last upgraded: aproximating an integer with a floating point. I wonder why. Maybe is worth checking if there is a bug upstream about this, and open one if not.